paasta_tools/paasta_maintenance.py

```diff
@@ -221,6 +222,8 @@ def paasta_maintenance():
     elif action == 'up':
         mesos_maintenance.up(hostnames)
     elif action == 'status':
+        ret = "%s" % mesos_maintenance.friendly_status()
```

Comment:
why do you need to do this interpolation? isn't ``ret = mesos_maintenance.friendly_status()`` enough?